restapi/src/main/java/io/stargate/web/resources/TableResource.java

```diff
@@ -200,13 +200,15 @@ public Response addTable(
           db.getAuthorizationService()
               .authorizeSchemaWrite(token, keyspaceName, tableName, Scope.CREATE, SourceAPI.REST);
 
+          int ttl = (options == null) ? 0 : options.getDefaultTimeToLive();
```

Comment:
`getDefaultTimeToLive()` returns an object `Integer`... Would it make sense to cover the case when it is `null`? It seems to be `null` by default.

Reply:
Good point. IMHO, we could 
1) Set a default value
```
public class TableOptions {

  private Integer defaultTimeToLive = 0;
```  
as both the constructor and the setter  (shown below) currently accepts the primitive type only, so they cannot set `defaultTimeToLive` to null. 

```
  public TableOptions(
    @JsonProperty("defaultTimeToLive") final int defaultTimeToLive,
    ...
    public void setDefaultTimeToLive(int defaultTimeToLive) {
      this.defaultTimeToLive = defaultTimeToLive;
     }
    ...
  ```
  
  2) Make `defaultTimeToLive` field a primitive type for the same reasons as above;
  
  I am more inclined to the second option. Wdyt?

Reply:
It's defined as an Integer so that we can validate during update without accidentally setting TTL to 0 when it's not intended
https://github.com/stargate/stargate/blob/master/restapi/src/main/java/io/stargate/web/resources/v2/schemas/TablesResource.java#L326-L332

Since it's not marked as required for createTable I think that if `defaultTimeToLive == null` then we just set it as 0 and create the table.

Reply:
Sure, makes sense. I have just done the changes as required to treat `defaultTimeToLive` as `null`. :)